src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/router/street/AccessEgressRouter.java

```diff
@@ -45,7 +45,7 @@ private AccessEgressRouter() {}
 
         NearbyStopFinder nearbyStopFinder = new NearbyStopFinder(
             rctx.graph,
-            rr.getMaxAccessEgressDuration(streetMode),
+            rr.getMaxAccessEgressDurationForMode(streetMode),
```

Comment:
FYI: In general, I do not think it is necessary to repeat the arguments in the name of the method, I know it is common for naming e.g. maps; hence also naming accessors for maps - but if it is clear form the context, then DRY is also ok. So, I think both names here are within the "authors license".

Reply:
I'm glad you say this because I think the `forMode` part is superfluous.